.github/actions/build-dependencies/action.yaml

```diff
@@ -20,7 +20,7 @@ runs:
         sudo apt-get install --yes pkgconf
         sudo apt-get install --yes llvm-12 clang-12
         sudo apt-get install --yes clang-format-12
-        sudo apt-get install --yes zlib1g-dev libelf-dev
+        sudo apt-get install --yes libelf-dev libzstd-dev zlib1g-dev
```

Comment:
I think you only really need `libelf-dev`, the rest is automatically pulled in:

* https://packages.ubuntu.com/noble/libelf-dev

Reply:
It's indeed. In fact, it could just be the main `libbpf-dev`, but it might be good to have them listed so that users from different environments (even tarball ones) can take a look at what is needed. @rscampos FYI.